src/ui/webbrowser/webbrowser.ios.ts

```diff
@@ -1,28 +1,36 @@
-import { AbstractWebBrowser } from './webbrowser';
+import { IWebBrowserOptions, WebBrowserBase } from './webbrowser';
 import Page from '../page';
-import Color from '../color';
 import Invocation from '../../util/iOS/invocation';
-import { WebBrowserOptions } from './webbrowseroptions';
 
-export default class WebBrowserIOS implements AbstractWebBrowser {
-  static readonly Options = WebBrowserOptions;
-
-  static show(page: Page, options: Partial<WebBrowserOptions>) {
-    if (!(options && options.url && (options.url.startsWith('https://') || options.url.startsWith('http://')))) {
+export default class WebBrowserIOS extends WebBrowserBase {
+  private _options?: Partial<IWebBrowserOptions>;
+  constructor(params?: Partial<IWebBrowserOptions>) {
+    super(params);
+    this._options = Object.assign({}, params);
```

Comment:
Is this going to be used? If so, the statement below about constructor parameters are invalid.